chain-storage/src/jellyfish.rs

```diff
@@ -230,6 +233,36 @@ fn decode_stale_node_index(data: &[u8]) -> Result<StaleNodeIndex> {
     })
 }
 
+/// Iterate through all stakings
+pub fn iter_stakings<S: GetKV>(
+    storage: &S,
+    version: Version,
+) -> impl Iterator<Item = StakedState> + '_ {
```

Comment:
in production, this will take a lot of time

Reply:
yes